### ISSUE TITLE
Unable to access github

### ISSUE THREAD

**Issue by eowyn7** [BME1478H/Winter2020class#156]
I am working on part c of the 2nd part of the assignment and typed
_**$ git push**_ into bash and the following was returned.

remote: Permission to BME1478H/Winter2020class.git denied to eowyn7.
fatal: unable to access 'https://github.com/BME1478H/Winter2020class.git/': The requested URL returned error: 403

What am I doing wrong?

**Comment by joelostblom** [comment]
You're trying to push to this repo rather than to your fork. Create a fork of this repo, push to to it, and then open a PR from your fork to this repo.

**Comment by eowyn7** [comment]
Can you please dumb that down?

**Comment by joelostblom** [comment]
Yes, sorry if it was unclear. You can view the GitHub step by step instructions for [how to create a fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)  (a version of this repo that you own) and [how to create a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) (a suggestion to incorporate changes from your fork into this repo). The pushing works as before, but since you have now cloned your fork rather than the main repo, you will have permission to pushto it and should not see the error.

Some more details to the steps I gave above:
Create a fork of this repo, clone it, make your changes locally (as before), push to the fork, and then open a PR from your fork to this repo via the GitHub interface.

**Comment by joelostblom** [comment]
Did this work for you @eowyn7 ?

**Comment by eowyn7** [comment]
yes this did, sorry i forgot to mention
